docs-website/pages/plugins-debug-plugin.md

```diff
@@ -1,11 +1,68 @@
 ---
-title: Dev Environment
-summary: Some preferences to make developing Plugins a bit more pleasant
-permalink: /guides/preferences/
-redirect_from: /introduction/preferences/
-order: 120
+title: Debug a plugin
+section: plugins
+chapter: Guides
+permalink: /plugins/debugging
+
+# Previous developer documentation
+redirect_from:
+  - /guides/debugging-plugins/
+  - /guides/preferences/
+  - /introduction/preferences/
+
+order: 103
 ---
 
+When developing a Sketch plugin, chances are you will need some ways to know what is happening when your code is running.
+
+## Logs
+
+The most common way to debug a JavaScript code is to put a bunch of `console.log` at key steps. Unfortunately, JavaScriptCore (the [context in which a Sketch plugin is running](/guides/cocoascript/)) doesn't provide `console`. Instead, a special method called `log` is available.
```

Comment:
That's not true anymore: `console.log` is polyfilled by Sketch

Reply:
I haven't touched any of the debugging content yet 🙂 Hopefully tomorrow